jib-core/src/main/java/com/google/cloud/tools/jib/image/ImageReference.java

```diff
@@ -256,6 +256,16 @@ public boolean usesDefaultTag() {
     return isDefaultTag(tag);
   }
 
+  /**
+   * Returns {@code true} if the {@link ImageReference} uses a SHA-256 digest as its tag; {@code
+   * false} if not.
```

Comment:
Maybe also add `When the tag is a digest, in string form, it is prepended with the at {@code @} symbol instead of a colon (@code :}.`